File: task 03/main.py
def pri(n):
    if n < 2:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n% i == 0:
            return False
    return True

try:
    n = int(input("Enter the number  "))
    print("Prime numbers up to the number is :")
    for i in range(2, n + 1):
        if pri(i):
            print(i)
except ValError:
    print("error")

File: task 03/main.rs
use std::io;

fn main() {
    println!("Enter a number (n):");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: u64 = input.trim().parse().expect("Invalid input. Please enter a valid number.");

    if n < 2 {
        println!("Prime numbers start from 2 and above.");
        return;
    }

    println!("Prime numbers up to {}:", n);
    for i in 2..=n {
        let mut is_prime = true;
        for j in 2..(i as f64).sqrt() as u64 + 1 {
            if i % j == 0 {
                is_prime = false;
                break;
            }
        }
        if is_prime {
            println!("{}", i);
        }
    }
}
